### DISCUSSION TITLE
[Request] Add the Auto import "button" to the playlists list view

### DISCUSSION THREAD

**Discussion by BobWs** [navidrome/navidrome#3060] [Ideas]
Hi,
Would you please consider to add the Auto import "button" to the list view of the playlist section (see screenshot)

![Screen_Shot 1](https://github.com/navidrome/navidrome/assets/5485820/a8d46e00-b99f-47ca-9d4f-f53258bd826b)

**Comment by caiocotts** [comment]
Hey @BobWs,

I've implemented the auto-import toggle as you suggested #3078. 
Please let me know what you think.

**Reply by BobWs** [reply]
> Please let me know what you think.

Thank you! will do...
